.github/workflows/helm-test.yaml

```
@@ -0,0 +1,31 @@
+name: Lint and Test Charts
+
+on: pull_request
+
+jobs:
+  lint-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Fetch history
+        run: git fetch --prune --unshallow
```

I'm pretty sure you can do [this](https://github.com/actions/checkout#fetch-all-history-for-all-tags-and-branches):
```suggestion
        uses: actions/checkout@v2
        run: |
           git fetch --prune --unshallow
```

-- I've made the same [change request upstream](https://github.com/helm/chart-testing-action/pull/33)

---

```
        uses: actions/checkout@v2
        run: |
           git fetch --prune --unshallow
```
!==

```
- uses: actions/checkout@v2
- run: |
    git fetch --prune --unshallow
```